src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -33,31 +34,65 @@
 
 public class JvmMonitor extends AbstractMonitor
 {
-  /**
+  /*
```

Comment:
Why we need this change? In other places we use `/**` and it's also standard for description comments.